server/webapp/WEB-INF/rails.new/app/presenters/api_v2/agent_representer.rb

```diff
@@ -83,6 +84,14 @@ def free_space
       end
     end
 
+    def errors
+      mapped_errors = agent.errors
+      if ip_errors = mapped_errors.delete('ipAddress')
+        mapped_errors['ip_address'] = ip_errors
+      end
+      mapped_errors
+    end
```

Comment:
@juhijariwala has a nice pattern for doing this —

```
class FooRepresenter
  error_representer('ipAddress' => 'ip_address')
end
```

The `error_representer` will take care of the translation, and defining the `errors` property that you declared above.

You may need to copy some code from `ApiV1::BaseRepresenter`